lib/loader.js

```diff
@@ -56,7 +55,7 @@ module.exports = {
   BossBar: require('./bossbar'),
   supportedVersions,
   testedVersions,
-  supportFeature
+  supportFeature: (version, feature) => require('minecraft-data')(version).supportFeature(feature)
```

Comment:
can't do that here, version is not known at the top level

Reply:
ah you're taking it as arg ok

Reply:
fixed